backend/index.js

```diff
@@ -0,0 +1,23 @@
+const app = require("express")();
+const server = require("http").createServer(app);
+const io = require("socket.io")(server, {
+    // Set up of CORS settings for socket.io server
+    cors: {
+        origin: [
+            "http://localhost:3000"
```

Comment:
Nice, do we want to consider moving this out to a configuration file :) ?

Reply:
Yeah we should probably do either that, or reading from environment variables. I'll just merge it as is for now and then we can decide the specifics of configuration in the future.